gradle.properties

```diff
@@ -16,7 +16,7 @@
 group=org.apache.kafka
 # NOTE: When you change this version number, you should also make sure to update
 # the version numbers in tests/kafkatest/__init__.py and kafka-merge-pr.py.
-version=0.11.0.0-SNAPSHOT
```

Comment:
Yeah, it was added via the transactions system tests PR, which is a bit confusing. Maybe we should remove it from this branch, but better to do it in a separate PR if you're OK with that.